koku/api/report/ocp/utils.py

```diff
@@ -0,0 +1,23 @@
+from decimal import Decimal
```

Comment:
We really shouldn't create utils files for a single function. I would put this in one of the `{cluster|node}_capacity` files, and just import it in the other.

Reply:
Personally, I think utils is a good place to highlight functions that we use in a multitude of places to perform a specific task. For example, this function is used in the following files:
1. cluster_capacity.py
2. node_capacity.py
3. query_handler.py (we use it to formulate the unused pieces for the query_sum)

For me part of the purpose of this file is to make it obvious that changes to this function impact other areas of the codebase. Whereas if I put it in the node_capacity file, its not clear that the function is used outside of node capacity unless you search the codebase. Granted this is really  just a preference of mine for how I think we should scaffold reusable code snip its. 

However, I'll follow your lead here if you feel strongly about removing the utils file.

Reply:
Common code in a shared module is a good way to keep the code base organized. Lots of `utils.py` on the other hand just makes it harder to find things. Since this function is used across multiple modules, it would make sense to have it in a common location. Maybe `koku/api/utils.py`? Or just `ocp/__init__.py`?

There's nothing _techincally_ wrong with keeping the function in one of the `{cluster|node}_capacity` files. It just makes for a funny looking import from a neighboring module. Purely aesthetic.

Reply:
Yeah, I didn't see a lot of instances of code being in the  `__init__.py` for this project. I agree a lot of `utils.py` files is a bit much, just wanted to call out its common code. 

I think for now a comment can just highlight that its common code and we can move this one along. Be would be nice to get it off my list of things I have to check each day.